Makefile

```diff
@@ -0,0 +1,59 @@
+RELEASE ?= `grep -1 -A 0 -B 0 '<version>' pom.xml | head -n 1 | awk '{print $1}' | sed -e 's/.*<version>//' | sed -e 's/<\/version>.*//'`
```

Comment:
With regards to make there is a maven plugin to create docker images, I haven't used it and don't think it should be added now.  However, it's an option for later.